spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2552,6 +2574,19 @@ def arm_joint_cmd_callback(self, data: JointState) -> None:
 
         self.spot_wrapper.arm_joint_cmd(**arm_joint_map)
 
+    def _gripper_joint_cmd_callback(self, data: JointState) -> None:
```

Comment:
Is this function called anywhere?

Reply:
hahaha nope. Thanks for that. I think this was from testing

Reply:
It's removed now. Sorry about that. I was originally going to do it not as a service but as a JointState thing but then.... yeah....